### PR TITLE
Update timezone to latest - 2021e

### PR DESCRIPTION
**Pull request by dhvaneshshah97**
This PR updates the following Region's timezones

- Denver: Nov 1 2037
- Chicago: Nov 1 2037
- Phoenix: Sep 21 4145
- Los Angeles: Nov 1 2037
- New York: Nov 1 2037
- Halifax: Nov 1 2037

[Story Reference](https://drivetime.tpondemand.com/restui/board.aspx#page=board/4843984347431366367&appConfig=eyJhY2lkIjoiODcyNDkzOEZDQkVENUZFQTYwMTVFRUJEODhFNjFERTIifQ==&boardPopup=userstory/120094/silent)

### CODE REVIEW

File: data/packed/us.json [hunk 1]
```diff
@@ -67,7 +66,6 @@
 		"America/Indiana/Knox|US/Indiana-Starke",
 		"America/Kentucky/Louisville|America/Louisville",
 		"America/Los_Angeles|US/Pacific",
-		"America/Los_Angeles|US/Pacific-New",
```
Comment (YizhiHong) [inline data/packed/us.json:70]: is this intended?

Comment (dhvaneshshah97) [reply data/packed/us.json:70]: Yes, this pacific-new is not present in 2021e. It was present though in 2017b. So that's why I removed it

**Review by jonknack0 — APPROVED** [review]
Unit tests: n/a
Code style: pass
Functional logic: pass
Business logic: pass

**Review by YizhiHong — APPROVED** [review]
Business Logic: pass

**Pull request merged**